src/api/Migrations/20250918142259_AddAlterationDegreeCode.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace BDMS.Migrations;
+#pragma warning disable CA1505
+
+/// <inheritdoc />
+public partial class AddAlterationDegreeCode : Migration
+{
+    /// <inheritdoc />
+    protected override void Up(MigrationBuilder migrationBuilder)
+    {
+        migrationBuilder.Sql(@"
+            INSERT INTO bdms.codelist(
+                schema_cli, code_cli, text_cli_en, text_cli_de, text_cli_fr, text_cli_it, order_cli
+            ) VALUES
+                ('alteration_degree', '', 'weathered', 'verwittert', 'altére', 'alterato', 11);
```

Comment:
Q: Müsste es nicht altéré heissen? (in der codelist gibts schon très altéré und peu altéré)